Release 2023.03.00 (#4012)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md

<ApiLifecycle access="ie" />

## March

### Monthly release 2023.03.0

| Change | Expected in Preview Orgs |
|---------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|

> | [Authenticator enrollment using the /authorize endpoint is GA in Production](#authenticator-enrollment-using-the-authorize-endpoint-is-ga-in-production)       |March 06, 2023   |

Reviewer: Our convention is to use the "first time in Preview org" date here. So typically that is the "EA in Preview" date.

| [OIDC Identity Providers private/public key pair support is GA](#oidc-identity-providers-private-public-key-pair-support-is-ga)                                |March 06, 2023   |

> | [API service integrations are GA in Preview](#api-service-integrations-are-ga-in-preview)                                                                      |March 06, 2023   |

Reviewer: ```suggestion
| [API service integrations are GA in Preview](#api-service-integrations-are-ga-in-preview)                                                                      |November 3, 2022   |
```
I was working on this one, so the 1st Preview date is Nov 3, 2022 for this feature.

> | [Log Streaming is GA in Production](#log-streaming-is-ga-in-production)                                                                                        |March 06, 2023   |

Reviewer: ```suggestion
| [Log Streaming is GA in Production](#log-streaming-is-ga-in-production)                                                                                        |March 30, 2022   |
```

| [OAuth 2.0 authentication for inline hooks is GA in Preview](#oauth-2-0-authentication-for-inline-hooks-is-ga-in-preview)                                      |March 06, 2023   |
| [Transactional verification with CIBA is GA in Preview](#transactional-verification-with-ciba-is-ga-in-preview)                                                |March 06, 2023   |
| [Improvements to self-service account activities for AD and LDAP users](#improvements-to-self-service-account-activities-for-ad-and-ldap-users)                |March 06, 2023   |
| [Honor force authentication support for SAML apps API](#honor-force-authentication-support-for-saml-apps-api)                                                  |March 06, 2023   |
| [OIN Manager support for Workflow Connector submission is GA in Preview](#oin-manager-support-for-workflow-connector-submission-is-ga-in-preview)              |March 06, 2023   |
| [Rate limit increased for Event Hooks](#rate-limit-increased-for-event-hooks)                                                                                  |March 06, 2023   |

> | [Bugs fixed in 2023.03.0](#bugs-fixed-in-2023-03-0)                                                                                                           |March 06, 2023   |

Reviewer: ```suggestion
| [Bugs fixed in 2023.03.0](#bugs-fixed-in-2023-03-0)                                                                                                           |March 08, 2023   |
```
Monthly is scheduled for deployment on March 8th. Please update all appropriate dates.

#### Improvements to self-service account activities for AD and LDAP users

Previously, the self-service unlock (SSU) and self-service password reset (SSPR) flows created unnecessary friction for AD and LDAP users. This feature enhancement introduces a seamless magic link experience in emails sent to unlock accounts and reset passwords. Users no longer need to provide consent when using the same browser. In addition, after successfully unlocking their account, clicking the email magic link counts towards the application's assurance policy. After the assurance requirements are met, the user is signed directly in to the application. These improvements are now GA in Preview. See [Customize email notifications](/docs/guides/custom-email/main/#use-vtl-variables). <!--OKTA-584526-->

> #### Honor force authentication support for SAML apps API

Reviewer: ```suggestion
#### Honor force authentication support for SAML Apps API
```

Previously, the **Honor Force Authentication** parameter 
(`honorForceAuthn`) could only be set from the 
[SAML 2.0 App Integration Wizard](https://help.okta.com/okta_help.htm?type=oie&id=csh-apps-aiw-saml). 
When this property is set to `true`, users are prompted for their credentials when a SAML request has the `ForceAuthn` attribute set to `true`. You can now set this property for your SAML app without using the app integration wizard. See the [SAML 2.0 settings parameters in the Apps API](/docs/reference/api/apps/#add-saml-2-0-authentication-application). <!--OKTA-550077-->

#### OIN Manager support for Workflow Connector submission is GA in Preview

> Okta [Workflows](https://www.okta.com/platform/workflows/) is a no-code, if-this-then-that logic builder that Okta orgs can use to automate custom or complex employee onboarding and offboarding flows in your application. You can now publish Workflow connectors that you create with the [Workflows Connector Builder](https://help.okta.com/okta_help.htm?type=wf&id=ext-connector-builder)  in the Okta Integration Network (OIN) catalog. Publishing a Workflows Connector with Okta allows your customers to deeply integrate your product with all other connectors in the catalog. Submit your Workflow Connector by using the OIN Manager. See [Submit an integration](/docs/guides/submit-app/wfconnector/main/#submit-an-integration) for Workflows connectors.. <!--OKTA-571943 & OKTA-573202-->

Reviewer: ```suggestion
Okta [Workflows](https://www.okta.com/platform/workflows/) is a no-code, if-this-then-that logic builder that Okta orgs can use to automate custom or complex employee onboarding and offboarding flows in your application. You can now publish Workflow connectors that you create with the [Workflows Connector Builder](https://help.okta.com/okta_help.htm?type=wf&id=ext-connector-builder) in the Okta Integration Network (OIN) catalog. Publishing a Workflows connector with Okta allows your customers to deeply integrate your product with all other connectors in the catalog. Submit your Workflow connector by using the OIN Manager. See [Submit an integration](/docs/guides/submit-app/wfconnector/main/#submit-an-integration) for Workflows connectors. <!--OKTA-571943 & OKTA-573202-->
```
Sorry for the inconsistent capitalization of "Connectors". That was my fault when I was trying to figure out if that's a product that we need to capitalize.

#### Rate limit increased for Event Hooks

The number of events that can be delivered to Event Hooks is now 400,000 events per org, per day. See [Rate limits](https://developer.okta.com/docs/concepts/event-hooks/#rate-limits). <!--OKTA-573847-->

#### Bugs fixed in 2023.03.0

* When an admin used a group limit in an expression that was greater than 100 (for example, `Groups.startsWith("active_directory","",500)` ), /userinfo endpoint requests failed. (OKTA-576414)

## February

### Weekly release 2023.02.2

| Change | Expected in Preview Orgs |
|------------------------------------------------------------------------------------------------------------------|-----------------|

> | [Bugs fixed in 2023.02.2](#bugs-fixed-in-2023-02-2)                                                              |February 27, 2023 |

Reviewer: The release calendar says this release was deployed on March 2, 2023?

Author: Whoops.
